frontend/src/views/IntegrationsPage/components/IntegrationsSection/components/IntegrationsTable.tsx

```diff
@@ -0,0 +1,415 @@
+import { useMemo, useState } from "react";
```

Comment:
what do you think about making the search bar filter against the destination fields as well? I was under the impression that I can use the search bar to filter baesd on the destination values

<img width="1320" alt="image" src="https://github.com/user-attachments/assets/030c8733-85da-49f7-b972-a2e90b5e06d9">